js/mocha-tests/test/test-setup.js

```diff
@@ -0,0 +1,12 @@
+// needed for regenerator-runtime
+import 'babel-polyfill';
+
+import chai, { expect } from 'chai';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+
+chai.use(sinonChai);
+
+chai.config.truncateThreshold = 0;
```

Comment:
This allows to have full stack traces and objects printed in the console when test fails. It makes debugging/investigating failing tests easier (if you want to print objects for instance). I can add a comment to make this more explicit.